### ISSUE TITLE
RCT1 .sea files from RCTC show up when RCT1 integration is enabled.

### ISSUE THREAD

**Issue by IntelOrca** [OpenRCT2/OpenRCT2#17949]
Currently, if you set your RCT2 base path to RCT Classic, and also set an RCT1 base path to RCT1, you will get duplicate scenarios RCT1 scenarios appearing in the scenario list.

The .sc4 files from RCT1 are better since they are imported accurately, where as the .sea files are re-creations.

Simplest thing to do is to ignore any .sea scenario that matches an RCT1 base game if RCT1 integration is enabled.

**Comment by KatieZeldaKat** [comment]
> The .sc4 files from RCT1 are better since they are imported accurately

I'm not sure what you mean by this @IntelOrca, but if it's possible to fix the import of the .sea files, I would prefer these RCTC scenario files. The re-creations are not too different, but I prefer the additional scenery and modifications to make there not be a combo of pay-per-ride and pay-per-entry (as is in RCT2 vanilla).

Perhaps having an option to choose which game to filter out would be ideal. Assuming this is viable, of course.

**Comment by jensj12** [comment]
> > The .sc4 files from RCT1 are better since they are imported accurately
> 
> I'm not sure what you mean by this @IntelOrca, but if it's possible to fix the import of the .sea files, [...]

The .sea files are imported properly into OpenRCT2, but they are themselves inaccurate w.r.t. the RCT1 scenarios they represent.

**Comment by IntelOrca** [comment]
@KatieZeldaKat Those are very small fixes that could be applied on top of your SC4 imported scenarios either manually or by plugin. I am not sure how many players would want that feature to justify it being an in-game option.

**Comment by IntelOrca** [comment]
I don't think we currently ignore SC6 files. You should be have them in your user scenario folder rather than the actual base game directories. You should avoid changing those.

**Comment by KatieZeldaKat** [comment]
Oh yeah, I didn't mean that scenarios in general would be in the base game directories (I do have quite a few custom scenarios in the user scenario folder). I just tested having a scenario of sc0.sc6 inside of RCT1, which is recognized as Forest Frontiers in the scenario menu. This can potentially accomplish the goal customizing the base game scenarios without having development on it.

Personally though, I've just moved the original scenarios to a different folder so they don't appear twice in the menus and I can still play the SEA ones I prefer (minus Urban Jungle which I believe is the only original game scenario not ported to RCTC). Because of this workaround, I don't see a reason to have an in-game toggle like I suggested previously.

**Comment by IntelOrca** [comment]
Where a file is does not affect how it gets treated. Whether your scenario is in the RCT1 base directory or your user directory should not change anything. For your scenarios to be recognised as original replacements, they need to have the same name.
